content/en/writing/foreign-income.md

```diff
@@ -0,0 +1,57 @@
+
+
+### 1. Goods and Services Tax (GST)
+
+- **Threshold**: Acquiring a GST number becomes almost mandatory once your income surpasses some X lakhs INR in a given financial year. This step legitimizes your income and ensures compliance with Indian taxation laws. The X limits keeps changing and I'd recommend you check the latest one with a CA and NOT rely on Google for this.
+
+- **Sole Proprietorship**: If you are an individual, you get a GST number as a sole proprietor. This is the simplest way to get a GST number and is recommended for most consultants. You can also get a GST number as a private limited company, but that might be overkill for most consultants. My CA also recommended that I get an Udyog Certificate, so I did get one but I'm not sure if it's necessary.
+  
+- **Quarterly Filing**: If you are a sole proprietor or operating as an individual consultant, quarterly GST filing can be more manageable than doing it monthly. This approach helps streamline administrative tasks and keeps compliance straightforward.
+
+- **Accounting & Compliance**: Using a CA to file for managing both your GST and Letter of Undertaking (LUT) is advisable. It serves as an effective tool for official documentation and ensures a smoother filing process.
+  
+- **Current Account**: Having a current account separate from your personal finances helps in meticulous financial management. It allows you to distinctly separate your business income and expenses, making accounting more cleaner. E.g. you can use this current to pay for your AWS bills. This can help you save on GST, as you can claim GST input credit on these expenses.
+
+- **Letter of Undertaking (LUT)**: For consultants involved in international transactions, an LUT is crucial. This legal document clarifies that all relevant taxes will be paid exclusively in India, thereby simplifying compliance requirements for cross-border business. It is advisable to get an LUT even if you are not sure if you will be involved in international transactions. You need to mention the LUT in your invoices to foreign entities.
+
+- **GST on Foreign Transactions**: GST is not applicable on foreign transactions. However, you will need to mention the LUT in your invoices to foreign entities. This is to ensure that the GST authorities are aware that you are not liable to pay GST on these transactions.
+
+### 2. Section 44ADA
+
+- **50% Rule**: Section 44ADA offers a benefit by allowing you to assume 50% of your income as an expense for tax filing. This helps significantly in reducing your taxable income, which can be advantageous for many consultants. So if you make 100 lakhs INR in a year, you can assume 50 lakhs INR as your expenses and pay taxes on the remaining 50 lakhs INR. Assume 30% tax rate, you'd pay 15 lakhs INR in taxes. Effectively, you'd be paying 15% of your income in taxes. 
```

Comment:
Afaik, Section 44ADA is applicable only till 75 Lakhs as freelance income. Furthermore, it was only 50 Lakhs in FY2022-23. 

Or am I missing something here?

Reply:
Also, there's a surcharge (15% iirc) on crossing 1Cr.